data/com/mojang/blaze3d/systems/RenderPass.mapping

```diff
@@ -2,30 +2,61 @@ CLASS com/mojang/blaze3d/systems/RenderPass
 	METHOD bindSampler (Ljava/lang/String;Lcom/mojang/blaze3d/textures/GpuTexture;)V
 		ARG 1 name
 		ARG 2 texture
+	METHOD bindSampler (Ljava/lang/String;Lcom/mojang/blaze3d/textures/GpuTextureView;)V
+		ARG 1 name
+		ARG 2 texture
 	METHOD draw (II)V
 		ARG 1 firstIndex
 		ARG 2 indexCount
 	METHOD drawIndexed (II)V
 		ARG 1 firstIndex
 		ARG 2 indexCount
+	METHOD drawIndexed (III)V
+		ARG 1 firstIndex
+		ARG 2 indexCount
+		ARG 3 primCount
+	METHOD drawIndexed (IIII)V
+		ARG 1 firstIndex
+		ARG 2 index
+		ARG 3 indexCount
+		ARG 4 primCount
 	METHOD drawMultipleIndexed (Ljava/util/Collection;)V
 		ARG 1 draws
 	METHOD drawMultipleIndexed (Ljava/util/Collection;Lcom/mojang/blaze3d/buffers/GpuBuffer;Lcom/mojang/blaze3d/vertex/VertexFormat$IndexType;)V
 		ARG 1 draws
 		ARG 2 buffer
 		ARG 3 indexType
+	METHOD drawMultipleIndexed (Ljava/util/Collection;Lcom/mojang/blaze3d/buffers/GpuBuffer;Lcom/mojang/blaze3d/vertex/VertexFormat$IndexType;Ljava/util/Collection;)V
+		ARG 1 draws
+		ARG 2 buffer
+		ARG 3 indexType
+		ARG 4 uniforms
```

Comment:
dynamic uniform names

Reply:
This is not resolved. There are two different sets of uniforms during a drawMultiple call. This specifies only the ones that will be dynamically set per draw call through the UniformUploader and bypasses validation on them being set ahead of time. This should _not_ have all uniforms passed into it as just "uniforms" implies. this is "uniforms that you will set via UniformUploader ahead of at least the first draw but probably for all of them individually and its ok if they aren't set yet"